operators/dream_texture.py

```diff
@@ -87,20 +83,16 @@ def step_progress_update(self, context):
         bpy.types.Scene.dream_textures_progress = bpy.props.IntProperty(name="Progress", default=0, min=0, max=context.scene.dream_textures_prompt.steps + 1, update=step_progress_update)
         bpy.types.Scene.dream_textures_info = bpy.props.StringProperty(name="Info", update=step_progress_update)
```

Comment:
These properties are also needed in upscale.py. When the subprocess gets initiated from upscale the ui doesn't automatically update when these are changed until this operator runs.